decidim-core/app/types/decidim/core/user_type.rb

```diff
@@ -27,6 +27,10 @@ module Core
         resolve ->(user, _args, _ctx) { user.presenter.profile_path }
       end
 
+      field :disabledNotifications, !types.String, "The user's disabled notifications status" do
```

Comment:
```suggestion
      field :directMessagesEnabled, !types.String, "If the user making the request is logged in, it will return whether this recipient accepts a conversation or not. It will return false for non-logged requests." do
```

Can we call this field something like this? I think the current name is misleading, we are not talking about notifications here  but messages.
ALSO: the output of this field should be a boolean, message should be handled at application level (i18n content can be passed via `data-` attributes in html). Returning false if direct messages from the current logged user aren't accepted. Also returning false if there's no user logged in (because you can't send messages anyway if you are not logged in, and we shouldn't give this information for free). Finally, return true if accepts conversation with the current user.